examples/src/examples/VaultEngineSingleUnitExample.java

```diff
@@ -86,7 +83,9 @@ public static void main(String[] args) throws InterruptedException, JsonProcessi
       VaultCalculationsApi apiInstance = new VaultCalculationsApi(getApiClient());
       
       ApiResponse<Object> response = apiInstance.postAndCalculateWithHttpInfo(null, null, calcParameters);
-      
+      //Comment the above line and uncomment the below lines to add cache control configuration. Results are by default cached for 12 hours; Setting max-stale=300 will fetch a cached result which is 5 minutes older.
+      //String cache_control="max-stale=300";
+      //ApiResponse<Object> response = apiInstance.postAndCalculateWithHttpInfo(null, null, calcParameters);
```

Comment:
Do you need to pass `cache-control` argument?

Reply:
No need to pass the argument, I have updated the code with the arguement.